elixir/test/contracts_test.exs

```diff
@@ -126,6 +110,82 @@ defmodule Trento.ContractsTest do
       assert {:error, :unknown_event} =
                cloudevent |> CloudEvent.encode() |> Trento.Contracts.from_event()
     end
+
+    test "should decode additional attributes" do
+      event = %Test.Event{id: UUID.uuid4()}
+      time = DateTime.utc_now()
+      expiration = DateTime.add(time, 3, :minute)
+      time_attr = %Google.Protobuf.Timestamp{seconds: time |> DateTime.to_unix()}
+      expiration_attr = %Google.Protobuf.Timestamp{seconds: expiration |> DateTime.to_unix()}
+      user_id = 1
+
+      default_attributes = %{
```

Comment:
question: do we need to make a distinction here in the test?
The `attributes_from_event` function should return all the attributes, not matter if they are given by the user in the publishing or not.
I just find that having the looped scenario doesn't test anything new

Reply:
Fine for me f32862d3f736eb71a12ad297887fd61cd4e1bf57